apps/ui/components/applications/ApplicationsGroup.tsx

```diff
@@ -37,6 +37,12 @@ function ApplicationsGroup({
   if (roundPk == null) {
     return null;
   }
+  applications.sort((a, b) => {
+    return (
+      new Date(a.sentDate ?? "").getTime() -
+      new Date(b.sentDate ?? "").getTime()
```

Comment:
even though the `sentDate` is probably never empty `new Date(a.sentDate ?? 0)` is more correct.
`new Date("") === NaN` and `new Date(0) === 1970-01-01T00:00:00.000Z`